apps/src/templates/studioHomepages/JoinSectionArea.jsx

```diff
@@ -82,7 +82,7 @@ export default function JoinSectionArea({
         )}
         {isPlSections && joinedPlSections?.length > 0 && isTeacher && (
           <ParticipantSections
-            sections={joinedPlSections}
+            sections={joinedPlSections.concat(joinedStudentSections)}
```

Comment:
```suggestion
            sections={joinedPlSections.concat(joinedStudentSections ?? [])}
```

line 75 implies that `joinedStudentSections` technically could be undefined